tree/dataframe/test/dataframe_snapshot.cxx

```diff
@@ -1350,6 +1359,36 @@ TEST_F(RDFSnapshotArrays, MultiThreadJitted)
    ROOT::DisableImplicitMT();
 }
 
+TEST_F(RDFSnapshotArrays, RedefineArrayMT)
+{
+   static constexpr unsigned int newArraySize = 6u;
+   static_assert(kFixedSize != newArraySize);
+   ROOT::EnableImplicitMT(3);
```

Comment:
RAII struct that resets the IMT state and deletes the file?

Reply:
It's a copy of the ST test, made to look similar to the other MT tests in this file (see e.g. the context above). If we want to do it with RAII, it should probably be done for all the MT tests in this file.

Reply:
Let's leave it as is then